aiu_fms_testing_utils/utils/__init__.py

```diff
@@ -12,6 +12,9 @@
 
 def warmup_model(model: nn.Module, input_ids: torch.Tensor, max_new_tokens: int, **padding_kwargs):
     from torch_sendnn import torch_sendnn
+    import torch_sendnn
+
+    torch_sendnn.warmup_mode()
```

Comment:
this is not how this works. warmup_mode is a python context, so the code should look like this:

```python
    dprint("AIU warmup")
    pt_compile_model_time = time.time()
    extra_kwargs = {**padding_kwargs, "only_last_token": True}
    with torch_sendnn.warmup_mode():
        generate(model, input_ids, max_new_tokens=max_new_tokens, max_seq_len=model.config.max_expected_seq_len, use_cache=True, do_sample=False, contiguous_cache=True, extra_kwargs=extra_kwargs)
    pt_compile_model_time = time.time() - pt_compile_model_time
    dprint(f"PT compile complete, took {pt_compile_model_time:.3f}s")

    # All the code in lines 25-29 is no longer needed (update_lazyhandle is now implicit)
```